static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -151,6 +223,25 @@ define([
                 bean.on(el, 'submit', submitSignUpForm);
             });
 
+            var answersDeliveryPreferences = $('.btn-answer-delivery-' + atomId);
+
+            answersDeliveryPreferences.each(function (el) {
+                bean.on(el, 'click', function(event) {
+                    submitDeliveryPreference(event)
+                    this.classList.add('is-clicked')
```

Comment:
Ideally you want to remove all these listeners once the user has clicked.

Reply:
Would `bean.off(el, 'click')` be sufficient, or should I abstract out the anon function so that the handler is removed too?

Reply:
You'll have to turn _all_ event listeners off, which will be painful. I suggest you delegate to the parent element like this:
```js
bean.one(formElement, 'click', answersDeliveryPreferences, function(evt) {
  ...
})
```